trade-finance-manager-api/src/v1/controllers/deal.submit.controller.js

```diff
@@ -196,6 +200,14 @@ const submitMigratedDeal = async (dealId, dealType, checker) => {
       updatedDeal.manualInclusionNoticeSubmissionDate = portalMINUpdate.details.manualInclusionNoticeSubmissionDate;
       updatedDeal.checkerMIN = portalMINUpdate.details.checkerMIN;
     }
+
+    // ACBS interaction : AIN or MIN only
+    if (dealController.canDealBeSubmittedToACBS(portalMINUpdate.submissionType)) {
+      console.info('Migrated deal ACBS interaction initiated: ', dealId);
+      await sendMigratedDealEmail(dealId);
```

Comment:
where is this email going to?

Reply:
you need to specify the email address when you send a notification in Production env

Reply:
Me, you and James.

Reply:
Ok. But you need specify the email address in production otherwise it won't work